knowledge_repo/app/static/css/custom.css

```diff
@@ -256,29 +241,6 @@ a.label:focus, a.label:hover {
   color: #9CA299;
 }
 
-.feed-author {
-  padding-bottom: 5px;
```

Comment:
That's a fair point, and happy to comply. I was looking forward to the upcoming url revamp on the index routes whereby /feed becomes /index/feed (as documented earlier in an internal roadmap document). We also discussed briefly the fact that these alternative index views are potentially going to be deprecated by a more complete filtering interface, which would consolidate all of the index views into one /index route. The qualification to `index-post` instead of just `index` was supposed to allow other css classes to be present if required without clashing in the namespace. Perhaps I should have written all this in the PR description; but if you still convinced `feed-post` is a better name, I'll change them over. :).

As a side note, this won't make the diff look much different, since these css classes are defined in a separate css file (and only applied to the index/feed page to which they are relevant).